.github/workflows/_e2e_tests.yml

```diff
@@ -72,5 +72,7 @@ jobs:
           ADMIN_UVICORN_PORT: "8081"
           COMMON_LOG_LEVEL: "DEBUG"
           API_HF_TIMEOUT_SECONDS: "10"
+          API_HF_JWT_PUBLIC_KEY_URL: "https://huggingface.co/api/keys/jwt"
+          API_HF_JWT_ALGORITHM: "EdDSA"
```

Comment:
By the way you shouldn't need this, you should be able to read this from the JWK no?

Reply:
I followed the recommendation of https://pyjwt.readthedocs.io/en/stable/algorithms.html#specifying-an-algorithm:

> Do not compute the algorithms parameter based on the alg from the token itself, or on any other data that an attacker may be able to influence, as that might expose you to various vulnerabilities (see [RFC 8725 §2.1](https://www.rfc-editor.org/rfc/rfc8725.html#section-2.1)). Instead, either hard-code a fixed value for algorithms, or configure it in the same place you configure the key. Make sure not to mix symmetric and asymmetric algorithms that interpret the key in different ways (e.g. HS* and RS*).

Reply:
ok, if not from the JWT you should be able to read it from the JWK no?

Reply:
If I understand well the code of https://github.com/jpadilla/pyjwt/blob/7665aa625506a11bae50b56d3e04413a3dc6fdf8/jwt/api_jwk.py and https://github.com/jpadilla/pyjwt/blob/7665aa625506a11bae50b56d3e04413a3dc6fdf8/jwt/algorithms.py:
- yes: the algorithm is obtained from `alg` or `crv`
- but: the key (`EdDSA` in our case) is not exposed in [PyJWK](https://github.com/jpadilla/pyjwt/blob/7665aa625506a11bae50b56d3e04413a3dc6fdf8/jwt/api_jwk.py#L10) or it's field [Algorithm](https://github.com/jpadilla/pyjwt/blob/7665aa625506a11bae50b56d3e04413a3dc6fdf8/jwt/api_jwk.py#L50)
- and: that key is required when calling [jwt.decode](https://github.com/jpadilla/pyjwt/blob/7665aa625506a11bae50b56d3e04413a3dc6fdf8/jwt/api_jwt.py#L147)

Reply:
alright, it's not a blocker